ZohoCRMCharts/Classes/ZCRMComparatorCell.swift

```diff
@@ -0,0 +1,260 @@
+//
+//  ZCRMComparatorCell.swift
+//  ZohoCRMCharts
+//
+//  Created by Sarath Kumar Rajendran on 12/11/18.
+//  Copyright © 2018 Zoho CRM. All rights reserved.
+//
+
+import UIKit
+
+internal final class ZCRMComparatorCell: UICollectionViewCell, ZCRMLayoutConstraintDelegate {
+	
+	internal var type: ZCRMCharts.ZCRMComparatorType!
+	internal var options: ComparatorRenderOptions = ComparatorRenderOptions()
+	internal var isHeader: Bool = false
+	private let label: UILabel = UILabel()
+	private let container: UIView = UIView()
+	internal var viewConstraints: [NSLayoutConstraint] = []
+	internal var chunkData: ZCRMChunkData! {
+		didSet {
+			if chunkData != nil {
+				self.setData()
+			} 
+		}
+	}
+	internal var highValue: Int!
```

Comment:
revalidate the default unwrapping needs.

Reply:
👍🏻👍🏻👍🏻